### DISCUSSION TITLE
Adding options to select class="selectpicker" does not work (moved)

### DISCUSSION THREAD

**Discussion by idhamari** [twbs/bootstrap#36783] [General]
Moved to the [select forum](https://github.com/snapappointments/bootstrap-select/discussions/2765)

**Comment by TravelTimN** [comment]
Hi @idhamari 

[bootstrap-select](https://github.com/snapappointments/bootstrap-select) is a separate plugin used in conjunction with Bootstrap.
Your query is probably better suited for their [Discussion Board](https://github.com/snapappointments/bootstrap-select/discussions).

However, that being said, I've tried your code on codepen real quick, and I'm seeing both the X data and the Y data updated successfully. The `x` converts to `new` as you've outlined on the scripts. Check the screenshot for demo...

![image](https://user-images.githubusercontent.com/41592028/179958318-fd83b04e-d336-4f75-a673-81472a2910f8.png)

and the console tests you've added:

![image](https://user-images.githubusercontent.com/41592028/179959086-fcea1b9f-e5c4-4d81-8c72-f0f2edc9bfa4.png)

**Reply by idhamari** [reply]
@TravelTimN Thanks for the info. I will move the discussion to the suggested forum as I still have the error.
